### PR TITLE
Disable autorebase when behind base branch option for renovate

### PR DESCRIPTION
**Pull request by ahmed-nour-fdc**
* DIsable autorebase for renovate.
* Change schedule to be daily.

### CODE REVIEW

**Comment by hannesg** [comment]
Could you explain why we should not use autorebase?

**Comment by sven-urbanski-freiheit-com** [comment]
> Could you explain why we should not use autorebase?

It's just very spammy and mostly not necessary.
If we merge 1 change, N builds would be retriggered.

**Comment by ahmed-nour-fdc** [comment]
> Could you explain why we should not use autorebase?

Renovate should only run on a schedule to check wether there is a package that needs to be updated, but currently with `autorebase` it runs whenever a commit is merged which causes lots of emails to be sent for no real reason.
I think once per day should be sufficient.

**Review by hannesg — APPROVED** [review]

**Comment by hannesg** [comment]
Makes sense. LGTM!

File: renovate.json [hunk 1]
```diff
@@ -35,9 +35,7 @@
     "gomodUpdateImportPaths",
     "gomodTidy"
   ],
-  "rebaseWhen": "behind-base-branch",
```
Comment (sven-urbanski-freiheit-com) [inline renovate.json:38]: maybe we need to set this to "conflicted" excplicitely. Not sure about the default value

**Pull request merged**